lib/editor/components/timetable/TripSeriesModal.js

```diff
@@ -61,9 +62,18 @@ const TripSeriesModal = (props: Props) => {
           display: 'flex',
           flexDirection: 'row'
         }}>
-          <div>{messages('startTime')}<HourMinuteInput onChange={setStartTime} seconds={startTime} standaloneInput /></div>
-          <div style={{margin: '0px 0px 0px 20px'}}>{messages('headway')} <HourMinuteInput onChange={setHeadway} seconds={headway} standaloneInput /></div>
-          <div style={{margin: '0px 0px 0px 20px'}}>{messages('endTime')} <HourMinuteInput onChange={setEndTime} seconds={endTime} standaloneInput /></div>
+          <div>
+            {messages('startTime')
+            }<HourMinuteInput onChange={setStartTime} seconds={startTime} showSeconds={useSecondsInOffset} standaloneInput />
+          </div>
+          <div style={{margin: '0px 0px 0px 20px'}}>
```

Comment:
Is this resetting an existing margin? Otherwise it can just be margin-left

Reply:
☺️